### PR TITLE
Update dynamic-theme-fixes.config

### PR DESCRIPTION
**Pull request by chuxubank**
Fix developer.android.com

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -2541,6 +2541,25 @@ developer.android.com
...
+CSS
+[theme=android-theme] {
+    --devsite-header-color-upper: var(--darkreader-neutral-background);
+    --devsite-footer-background: var(--darkreader-neutral-background);
+    --devsite-search-form-background-active: var(--darkreader-neutral-background);
+    --devsite-searchbox-inactive: unset;
+    --devsite-searchbox-active: var(--darkreader-neutral-background);
+    --devsite-searchbox-hover: unset;
+}
+.devsite-wrapper {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:2553]: For the following 3 CSS entries. `Why?`

Comment (chuxubank) [reply src/config/dynamic-theme-fixes.config:2553]: Let's take https://developer.android.com/jetpack/compose/layout as an example:
1. `.devsite-wrapper` is for the strange background color change across the page. (Notice the gray background divider in the middle of the page.)
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
image
</td>
            <td>image</td>
            <td>image</td>
        </tr>
</table>

2. `button` background (Notice the thumb up and down icon button as well as the setting button.)
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
image</td>
            <td>
image</td>
            <td>
image</td>
        </tr>
</table>

3. `code` background (Notice the inline code background)
I have no idea why Dark Reader do not convert the background color for `code` block, and the color `silver` may not be the best color?
<table>
        <tr>
            <td align=center>Origin</td>
            <td align=center>Dark Reader</td>
            <td align=center>Fix</td>
        </tr>
        <tr>
            <td>
image
</td>
            <td>
image
</td>
            <td>
image</td>
        </tr>
</table>

**Comment by chuxubank** [comment]
I just notice the search bar is still white on Dark Mode, I will fix it.

**Comment by chuxubank** [comment]
image
@Gusted I'm done with the fix, could you please review it again? Thx!

**Review by Gusted — APPROVED** [review]
Thanks for the clarification looks good! Thank you!

**Pull request merged**